File: Exercicio3.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL,"");
    int a, b, c, d, dif;

    printf("Insira um valor: ");
    scanf("%d",&a);
    printf("\nInsira um valor: ");
    scanf("%d",&b);
    printf("\nInsira um valor: ");
    scanf("%d",&c);
    printf("\nInsira um valor: ");
    scanf("%d",&d);

    dif = a*b - c*d;
    printf("\n\nA diferença é igual a %d",dif);
    return 0;
}

File: Exercicio3.cs
using System;

class Program {
  public static void Main (string[] args)
        {
            int a, b, c, d, dif;

            Console.WriteLine("Insira um valor: ");
            a = int.Parse(Console.ReadLine());
            Console.WriteLine("Insira um valor: ");
            b = int.Parse(Console.ReadLine());
            Console.WriteLine("Insira um valor: ");
            c = int.Parse(Console.ReadLine());
            Console.WriteLine("Insira um valor: ");
            d = int.Parse(Console.ReadLine());

            dif = (a * b) - (c * d);
            Console.WriteLine("\r\nA diferenca eh igual a " + dif);

        }
}